sources/packages/web/src/components/common/StudentApplication.vue

```diff
@@ -19,6 +19,29 @@
     secondaryLabel="Previous section"
     class="mx-0"
   >
+    <template #primary-buttons v-if="isLastPage">
+      <span>
```

Comment:
To address the comment above and also not to have the span(line 23) created during a read-only load, 

following solution can be followed. 

Add the condition `!readonly` to line 22(template) and remove from line 34. The solution will look like this.

![image](https://github.com/user-attachments/assets/0539ebff-9c47-4062-873e-582bbeda6603)